src/scenarios/blazorlocalized/test.py

```diff
@@ -0,0 +1,23 @@
+'''
+Empty Blazor Wasm Template
+'''
+import os
+from shared.runner import TestTraits, Runner
+from shared.const import APPDIR
+
+EXENAME = 'blazorlocales'
+
+if __name__ == "__main__":
+    net5_linker_dump = os.path.join(APPDIR, 'obj', 'Release', 'net5.0', 'browser-wasm', 'linked', 'linker-dependencies.xml.gz')
```

Comment:
Are these still needed? Because either we fix the paths (here and in other Blazor scenarios) or we remove those.

cc @radical

Reply:
Do we care about "supporting" net5/net6 in these scenarios? If not, then yeah, we can remove these from all the scenarios.

Reply:
We don't, there's no hybrid globalization feature there so it does not make sense.